### PR TITLE
Switch to new native-image building

### PR DESCRIPTION
**Pull request by LeStarch**
This changes the native build to use the standard native image, and provides binary wheels.

### CODE REVIEW

File: compiler/test [hunk 1]
```diff
@@ -1,4 +1,8 @@
 #!/bin/sh
 
-export COMPILER_ROOT=$PWD
+# Fix when run from a differen directory
+wd=`dirname $0`
+cd $wd
+
+export COMPILER_ROOT=`cd $wd; pwd`
```
Comment (bocchino) [inline compiler/test:7]: Do we need the cd $wd in line 7? It looks redundant with line 5.

**Review by bocchino — COMMENTED** [review]
Looks good! Just one comment.

File: compiler/test (hunk 1 above)
Comment (LeStarch) [reply compiler/test]: Not needed, this is fixed!

**Review by bocchino — APPROVED** [review]
Looks great! It works because CI passes.

**Pull request merged**